packages/js/product-editor/src/components/editor/types.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { Product } from '@woocommerce/data';
+import {
+	EditorSettings,
+	EditorBlockListSettings,
+} from '@wordpress/block-editor';
+import { Template } from '@wordpress/blocks';
+
+export type LayoutTemplate = {
+	id: string;
+	title: string;
+	description: string;
+	area: string;
+	blocks: Template[];
```

Comment:
Since these are templates and not actual blocks, maybe they should be called `blockTemplates`?

Reply:
done